src/simulator.cpp

```diff
@@ -241,11 +336,80 @@ void Simulator::DumpResult() {
         std::cerr << "Error replacement setting" << std::endl;
         exit(-1);
     }
-    std::cout << "\n";
-    std::cout << "Number of cache access： " << _counter.access << std::endl;
-    std::cout << "Number of cache load： " << _counter.load << std::endl;
-    std::cout << "Number of cache store： " << _counter.store << std::endl;
-    std::cout << "Cache hit rate: " << std::setprecision(6)
-              << _counter.avg_hit_rate << std::endl;
-    std::cout << "===================================" << std::endl;
 }
+
+void Simulator::_DumpCACTIConfig(const std::string filename,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'filename' should be passed by const reference.](https://app.codacy.com/app/jerry0715no14/Cache_sim/pullRequest?prid=4646302)